ansible_ai_connect/ai/api/tests/test_formatter.py

```diff
@@ -599,11 +599,17 @@ def test_restore_original_task_names_for_playbook_with_task_preset(self):
             "    - name:  Install Apache\n      ansible.builtin.apt:\n        "
             "name: apache2\n        state: latest\n"
         )
-        multi_task_prompt = "# say hello test@example.com\n"
+        multi_task_prompt = "# Say hello & install mail server & email TO aykut@redhat.com \n"
 
         multi_task_yaml = (
-            "    - name:  say hello test@example.com\n      "
-            "ansible.builtin.debug:\n        msg: Hello there olivia1@example.com\n"
+            "    - name:  Say hello\n      "
+            "ansible.builtin.debug:\n        msg: Hello there\n"
+            "    - name: Install mail server\n      "
+            "ansible.builtin.package:\n        name: postfix\n        state: latest\n"
+            "    - name: Email TO aykut@redhat.com\n      "
```

Comment:
@mabulgu RH email can be replaced with some example email, correct?

Reply:
no harm keeping that one as I saw some examples but updated it anyways. pls recheck